pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -86,6 +89,11 @@ def __init__(
 
         # Initialize the C++ binds
         self._jacobian_lightning, self._create_ops_list_lightning = self._adjoint_jacobian_dtype()
+        
+        # Warning about performance with MPI and batch observation 
+        if self._use_mpi and self._batch_obs:
+            warn("It is not possible to use LGPU with MPI and batch the observation. batch_obs will be deactivated",RuntimeWarning)
```

Comment:
hmmm, `_batch_obs` in `lightning.gpu-mpi` mean loop all terms of a Hamiltonian `obs` one by one instead of one pass. This means `batch_obs =False` requires more memory and is faster, while `batch_obs = True` allows larger number of qubits simulation at the expense of high computational cost. Better to mention it explicitly in the `docstr`.